cli/tools/registry/pm.rs

```diff
@@ -32,10 +33,13 @@ use crate::jsr::JsrFetchResolver;
 use crate::npm::NpmFetchResolver;
 
 mod cache_deps;
+pub(crate) mod deps;
```

Comment:
FWIW, there's not much point to `pub(crate)` in the CLI crate. I remove it whenever I see it.